java/vector/src/main/codegen/templates/UnionListWriter.java

```diff
@@ -176,6 +176,29 @@ public StructWriter struct(String name) {
     return structWriter;
   }
 
+  @Override
+  public MapWriter map() {
+    return writer;
+  }
+
+  @Override
+  public MapWriter map(String name) {
+    MapWriter mapWriter = writer.map(name);
+    return mapWriter;
+  }
+
+  @Override
+  public MapWriter map(boolean keysSorted) {
+    writer.map(keysSorted);
```

Comment:
same here, return the new `mapWriter`?

Reply:
I responded above on why `mapWriter` isn't returned here.